modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,94 @@
+
+`ORG A` has one API Group called `Social Media` with one version `V1`. `Social Media V1` has 2 API Group instances:
+
+* API Group instance `QA` in `Sandbox` environment. This instance has 4 API instances: `Facebook v1` (QA) and `Twitter v1` (QA) from `ORG A` and `Instagram v1` (QA) and `Snapchat v1` (QA) from `ORG B`.
+* API Group instance `PROD` in `Production` environment. This instance has 4 API instances: `Facebook v1` (PROD) and `Twitter v1` (PROD) from `ORG A` and `Instagram v1` (PROD) and `Snapchat v1` (PROD) from `ORG B`. 
+
+The API Group instance `QA` is published and an asset called `API Group social media V1` is created. If the API Group instance `PROD` is published, no asset will be created. In this case, `PROD` will only be linked to the existing asset because the API Group `Social Media` and the version `v1` are the same.
+
+== Example Issue to Solve Using API Groups
+
+To better understand how API Groups facilitates API consumer engagement, consider the needs of a fictitious company named Good Weather, which forecasts atmospheric conditions for a given location and time by using meteorology. 
+
+Good Weather enables customers to access requested results using the following APIs:
+
+* Locations API
+* Forecast API
+* Current Conditions API
+* Indices API
+* Weather Alarms API
+* Alerts API
+* Imagery API
+* Tropical API
+* Translations API
+
+Currently, the process that customers must follow to access and use the APIs is not efficient, requiring:
+
+* Searching the Good Weather company portal to find appropriate APIs to actualize a business flow
++
+To determine the precise API they need, customers have to search a long list of APIs on the portal. This difficulty is compounded when developers modify the existing APIs or introduce additional APIs. 
+* Changing multiple API service plans individually to adopt a service-wide change 
++
+This complicates the task of upgrading or downgrading of services.
+* Working to access each element of an API when API access is approved
++
+The API packaging does not restrict access down to the individual call and data elements. This introduces security issues and unnecessary overhead.
+
+== API Groups Solution
+
+Using API Groups, Good Weather can package different APIs based on consumer requirements and scenarios, with specific SLAs that do not clash with the requirements for other consumers. Good Weather can quickly adapt and absorb API changes. Using API Groups: 
+
+* Good Weather can package different sets of APIs to target different consumers' requirements.
+* API consumers can access a collection of related APIs instead of individual APIs, simplifying asset discovery on the Good Weather portal. 
+* API Groups can enforce governance using SLA plans that can be modified with a single click.
+
+The following table illustrates the API Group packages that Good Weather created to meet their business requirements:
+
+*API Groups Packages for Weather Forecasting*
```

Comment:
We don't generally allow titles for graphic elements, do we?

Reply:
Done! Thanks!